src/Event.php

```diff
@@ -161,6 +168,8 @@ public function getId(): string
     }
 
     /**
+     * Gets the SDK identifier for the SDK package that generated this Event.
+     *
      * @return string
```

Comment:
```suggestion
     * @return string
     *
     * @internal
```
     Let's make these functions internal.